lib/AST/ASTVerifier.cpp

```diff
@@ -2649,7 +2649,7 @@ class Verifier : public ASTWalker {
         // If we are performing pack iteration, variables have to carry the
         // generic environment. Catching the missing environment here will prevent
         // the code from being lowered.
-        if (var->getTypeInContext()->is<ErrorType>()) {
+        if (var->getTypeInContext()->hasError()) {
```

Comment:
Any thoughts on the suggested comment change in https://github.com/apple/swift/pull/67594#discussion_r1418804474?

Reply:
Holly has already updated it in #70348!